upcloud/client/client.go

```diff
@@ -14,7 +14,7 @@ import (
 
 // Constants
 const (
-	DefaultAPIVersion = "1.3.4"
+	DefaultAPIVersion = "1.3.6"
```

Comment:
IMO: Feels weird to keep this up to date by hand. Though don't know how to implement this in any other way 😅

Reply:
This should be a variable that is set in release phase thru the linker options. Let's anyway address this in a separate PR